test/test_other.py

```diff
@@ -10386,7 +10386,9 @@ def test_minimal_runtime_code_size(self, test_name, js, compare_js_output=False)
                           '-lembind',
                           '-fno-rtti',
                           '-DEMSCRIPTEN_HAS_UNBOUND_TYPE_NAMES=0',
-                          '-sDYNAMIC_EXECUTION=0'
+                          '-sDYNAMIC_EXECUTION=0',
+                          '-lc++',
+                          '-lc++abi'
```

Comment:
Why are these two libraries needed? They should be automatically included by the compiler.

Reply:
I'm not sure but I think it's because we use `-sSTRICT`. Without it I'd get linking errors for C++ builtins, and I think I've seen the same in the past when using `STRICT`.

Reply:
Ah.. in that case the fix is to use EMXX below instead of EMCC.

Reply:
Ah so it's not the AUTO_NATIVE_LIBRARIES that's causing the issue (I thought libc++ was counted among them) but the default C vs C++.

Is it safe to change it to EMXX even for other tests or does it have to be only for Embind?

Reply:
Ideally C tests should use `emcc` and C++ tests should use `em++`.

Reply:
For most tests this is already automatic.  See https://github.com/emscripten-core/emscripten/blob/054bf307ee8391ffea9037761e3c9e2bc54dcbbe/test/common.py#L946

In this case its enough to simply switch to EMXX I think.

Reply:
Alternatively you could use `self.emcc` or `self.build` instead of directly using `self.run_process`

Reply:
It's a bit confusing because e.g. in `hello_webgl_sources` there are two filenames, one is C++ and another is C, and modifying test means I modify it for all those testcases.

I guess I can just split out first argument and assume that's what should be used as a filename for now...

Reply:
OTOH switching to EMXX seems easier.

Reply:
Ok no I tried those different options and I'm just seeing different errors each time, mostly due to that mix of C and C++ in one invocation in hello_webgl_sources. I believe the way that test is done is not truly correct if we want to use different compilers for different langs, since technically each source would have to be compiled with `emcc`/`em++` respectively and then linked together... but I don't want to take on fixing that as part of this PR.

I'd rather keep the current workaround until someone refactors the parent test to account for different languages.